src/plain/number/probability.js

```diff
@@ -69,3 +69,30 @@ export const gammaP = [
   -0.26190838401581408670e-4,
   0.36899182659531622704e-5
 ]
+
+export function gammalnNumber (n) {
+  if (n < 0) return NaN
+  if (n === 0) return Infinity
+  if (!isFinite(n)) return n
+
+  const lnSqrt2PI = 0.91893853320467274178
+  const gammaSeries = [
+    76.18009172947146,
+    -86.50532032941677,
+    24.01409824083091,
+    -1.231739572450155,
+    0.1208650973866179e-2,
+    -0.5395239384953e-5
+  ]
```

Comment:
Do I need to make these two variables global? The used memory won't get GC but `gammalnNumber` can be about 12% faster.